clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -794,6 +796,13 @@ void BackendConsumer::DontCallDiagHandler(const DiagnosticInfoDontCall &D) {
                               ? diag::err_fe_backend_error_attr
                               : diag::warn_fe_backend_warning_attr)
       << llvm::demangle(D.getFunctionName()) << D.getNote();
+
+  SmallVector<std::string> InliningDecisions = D.getInliningDecisions();
+  InliningDecisions.push_back(D.getCaller().str());
+  for (auto [index, value] : llvm::enumerate(InliningDecisions))
```

Comment:
```suggestion
  for (const auto &[index, value] : llvm::enumerate(InliningDecisions))
```

Seems we don't modify either, and `value` is a string, so requiring a copy of the pair is expensive.

Reply:
done in d3a221a9aea4f60244adc7d726b398abeb2775b3 PTAL